targets/targets.json

```diff
@@ -2589,6 +2589,28 @@
             "STM32G431xx"
         ]
     },
+    "NUCLEO_G431KB": {
+        "inherits": [
+            "MCU_STM32G431xB"
+        ],
+      "overrides": {
+        "clock_source": "USE_PLL_HSI",
+        "lse_available":  0
+      },
+      "extra_labels_add": [
+        "STM32G431KB"
+      ],
+      "macros_add": [
+        "STM32G431KB"
```

Comment:
Same question, not needed ?

Reply:
Which line? extra_lables_add? Yeah, I suppose that can be removed - it's was present in the 6.3.0 mbed-os release but I see now that how it's used has been modified.

I'll go ahead and remove it.